test/unit/query-generator/add-column-query.test.ts

```diff
@@ -0,0 +1,34 @@
+import { DataTypes } from '@sequelize/core';
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#addColumnQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+  }, { timestamps: false });
+
+  it('generates a ADD COLUMN query in supported dialects', () => {
+    const sql = queryGenerator.addColumnQuery(User.tableName, 'age', {
+      type: DataTypes.INTEGER,
+    });
+
+    expectsql(() => sql, {
+      default: `ALTER TABLE [Users] ADD [age] INTEGER;`,
+      mssql: `ALTER TABLE [Users] ADD [age] INTEGER NULL;`,
+      postgres: `ALTER TABLE "public"."Users" ADD COLUMN "age" INTEGER;`,
+    });
+  });
+
+  it('generates a ADD COLUMN IF NOT EXISTS query in supported dialects', () => {
+    expectsql(() => queryGenerator.addColumnQuery(User.tableName, 'age', {
```

Comment:
Why is this not wrapped in a constant as well?

Reply:
Actually `addColumnQuery` throws an error when `ifNotExists` is passed to `options` therefore if I were to assign it to a constant the error would be thrown there.

```js
const sql = queryGenerator...
```

However to get the proper `default` behavior we invoke and expect it to equal `buildInvalidOptionReceivedError` call.